Userland/DevTools/HackStudio/Dialogs/ProjectTemplatesModel.cpp

```diff
@@ -21,12 +23,23 @@ ProjectTemplatesModel::ProjectTemplatesModel()
     : m_templates()
     , m_mapping()
 {
-    auto watcher_or_error = Core::FileWatcher::watch(ProjectTemplate::templates_path());
+    auto watcher_or_error = Core::FileWatcher::create();
     if (!watcher_or_error.is_error()) {
         m_file_watcher = watcher_or_error.release_value();
         m_file_watcher->on_change = [&](auto) {
             update();
         };
+
+        auto watch_result = m_file_watcher->add_watch(
+            ProjectTemplate::templates_path(),
+            InodeWatcherEvent::Type::ChildCreated
+                | InodeWatcherEvent::Type::ChildDeleted);
+
+        if (watch_result.is_error()) {
+            warnln("Unable to watch templates directory, templates will not automatically refresh. Error: {}", watch_result.error());
+        } else if (watch_result.value() != true) {
+            warnln("Unable to watch templates directory, are we already watching it?");
```

Comment:
Since this is a user-facing string, let's try to sound a bit more authoritative.
Why can't we report the specific kernel error that occurred?

Reply:
`watch_result` being false only happens when the filepath passed is not a valid `LexicalPath` or we are already watching it. The kernel error is returned via the `error` method instead. What should it say?